### ISSUE TITLE
Could not find module `ember-testing` imported from `@ember/test/index`

### ISSUE THREAD

**Issue by elwayman02** [embroider-build/embroider#805]
I'm seeing this error when deploying my addon's demo app to production after upgrading from 0.37 to 0.40 of `@embroider/test-setup`: https://github.com/elwayman02/ember-user-activity/pull/419

Here's a live reproduction of the error, which can be seen in the browser console: https://ember-user-activity.jhawk.co/

```
Uncaught Error: Could not find module `ember-testing` imported from `@ember/test/index`
    at vendor-6e50465db379986591a65cb5508c4ddc.js:13
    at c (vendor-6e50465db379986591a65cb5508c4ddc.js:13)
    at s.findDeps (vendor-6e50465db379986591a65cb5508c4ddc.js:25)
    at c (vendor-6e50465db379986591a65cb5508c4ddc.js:13)
    at s.findDeps (vendor-6e50465db379986591a65cb5508c4ddc.js:25)
    at c (vendor-6e50465db379986591a65cb5508c4ddc.js:13)
    at s.findDeps (vendor-6e50465db379986591a65cb5508c4ddc.js:25)
    at c (vendor-6e50465db379986591a65cb5508c4ddc.js:13)
    at s.findDeps (vendor-6e50465db379986591a65cb5508c4ddc.js:25)
    at c (vendor-6e50465db379986591a65cb5508c4ddc.js:13)
```

If I rollback to 0.37, the app works as expected.

**Comment by rwjblue** [comment]
Fixed by https://github.com/emberjs/ember.js/pull/19540 and released in 3.27.1.

**Comment by elwayman02** [comment]
So, just to clarify - even though this issue occurred on Ember 3.26 and was introduced by upgrading to `@embroider/test-setup` 0.40, the fix is to upgrade Ember? Does that mean the latest version of `@embroider/test-setup` is incompatible with Ember 3.26?
